setup.py

```diff
@@ -27,7 +27,7 @@
 # fmt: off
 setup(
     name='bluetooth-mesh',
-    version='0.2.13',
+    version='0.2.14',
```

Comment:
Perhaps it's worth bumping the minor as well, since the user-visible API has changed.

Reply:
Changes to `Model.unsubscribe` should be backwards-compatible.